### PR TITLE
Add pytest output

### PR DESCRIPTION
**Pull request by dcdarrell9**
## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by dcdarrell9** [comment]
[sdx-sequence - implement py.test](https://trello.com/c/A9mzoDm5/499-sdx-sequence-implement-pytest)

File: requirements.txt [hunk 1]
```diff
@@ -283,8 +283,8 @@ reportlab==3.4.0 \
 requests==2.12.4 \
     --hash=sha256:000748df49e087784441b2621c50fb81046c5c8e80e0d91674ffad65b9e13844 \
     --hash=sha256:ed98431a0631e309bb4b63c81d561c1654822cb103de1ac7b47e45c26be7ae34
-SQLAlchemy==1.0.12 \
-    --hash=sha256:6679e20eae780b67ba136a4a76f83bb264debaac2542beefe02069d0206518d1
+SQLAlchemy==1.1.14 \
```
Comment (warrenbailey) [inline requirements.txt]: Any reason why we updated SQLAlchemy as part of this PR?

File: tests/test_sequence.py [hunk 2]
```diff
@@ -91,26 +91,24 @@ def test_increment_wraps_image_sequence(self):
...
-    sequence_range = 1000000000
-
-    test_start = 1
-    test_end = 10
-
-    for i in range(test_start, test_end):
-        expected_seq_no = i % sequence_range
-        self.mock_sequence_response(self.json_sequence_endpoint, i, sequence_start, sequence_range, expected_sequence_no=expected_seq_no)
+        test_start = 1
+        test_end = 10
```
Comment (warrenbailey) [inline tests/test_sequence.py:99]: Why has this changed? Is it just formatting? If so, what was wrong with it?

Comment (dcdarrell9) [reply tests/test_sequence.py:99]: These weren't in the `TestSequenceService` class and they were failing, but passed when moved in

File: requirements.txt (hunk 1 above)
Comment (dcdarrell9) [reply requirements.txt]: My bad, tried a change elsewhere and hashin added it here, will remove

File: .gitignore [hunk 3]
```diff
@@ -9,3 +9,4 @@ lib/
 pip-selfcheck.json
 coverage.xml
 .coverage
+htmlcov/
```
Comment (gemmai95) [inline .gitignore]: Get rid of the no newline icon

**Review by gemmai95 — APPROVED** [review]

**Review by warrenbailey — APPROVED** [review]

**Pull request merged**